osu.Game.Rulesets.Osu/Edit/Blueprints/OsuSelectionBlueprint.cs

```diff
@@ -1,19 +1,29 @@
 // Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
+using osu.Framework.Allocation;
 using osu.Game.Rulesets.Edit;
+using osu.Game.Rulesets.Objects;
+using osu.Game.Rulesets.Osu.Edit.Blueprints.HitCircles.Components;
 using osu.Game.Rulesets.Osu.Objects;
 using osu.Game.Rulesets.Osu.Objects.Drawables;
+using osu.Game.Screens.Edit;
 
 namespace osu.Game.Rulesets.Osu.Edit.Blueprints
 {
     public abstract class OsuSelectionBlueprint<T> : HitObjectSelectionBlueprint<T>
         where T : OsuHitObject
     {
+        [Resolved]
+        private EditorClock editorClock { get; set; }
```

Comment:
From the state of CI it looks like this either needs to be optional, or some tests are going to need this cached.